source/community/contribution-guideline/index.md

```diff
@@ -12,105 +12,6 @@ group: Community
 <div class="toc-list"></div>
 
 ## Introduction
-First of all, thank you! You have decided to contribute code to our software and become a member of the large shopware community. We appreciate your hard work and want to handle it with the most possible respect.
+First of all, thank you that you have decided to contribute to shopware. 
```

Comment:
maybe **S**hopware **5** because the software is meant?

Reply:
Done =)